wamv_gazebo/urdf/ball_shooter/ball_shooter.xacro

```diff
@@ -45,7 +45,7 @@
           <frame>wamv/ball_shooter_link</frame>
           <pose>0.14 0 0.09 0 0 0</pose>
         </projectile>
-        <shot_force>250</shot_force>
+        <shot_force>300</shot_force>
         <topic>${namespace}/${shooter_namespace}${name}/fire</topic>
```

Comment:
As mentioned in general comments I think we also need
```
<num_shots> 4</num_shots>
```
to align with our documentation for this task.